lib/restforce/extensions.rb

```diff
@@ -10,8 +10,11 @@ class SObject
     # Raises on update error.
     def update!(attributes)
       ensure_id
-      @client.update!(sobject_type, attributes.merge("Id" => self.Id))
```

Comment:
What a bummer that `update!` doesn't return a usable result object instead of `true`.